tests/components/netatmo/test_api.py

```diff
@@ -0,0 +1,14 @@
+"""The tests for the Netatmo oauth2 api."""
+from unittest.mock import patch
+
+from homeassistant.components.netatmo import api
+
+
+async def test_api(hass, config_entry):
+    """Test auth instantiation."""
+    with patch(
+        "homeassistant.helpers.config_entry_oauth2_flow.async_get_config_entry_implementation",
+    ) as fake_implementation:
+        auth = api.ConfigEntryNetatmoAuth(hass, config_entry, fake_implementation)
+
+    assert isinstance(auth, api.ConfigEntryNetatmoAuth)
```

Comment:
This test doesn't add much value I think. We're asserting a detail of the integration which we try to avoid normally. Instead we should test that we can set up the integration, while patching the library or using a request mocker and that core state is the expected.